doc/classes/ProjectSettings.xml

```diff
@@ -386,7 +386,15 @@
 			[b]Note:[/b] If the operating system blocks access to audio input devices (due to the user's privacy settings), audio capture will only return silence. On Windows 10 and later, make sure that apps are allowed to access the microphone in the OS' privacy settings.
 		</member>
 		<member name="audio/driver/mix_rate" type="int" setter="" getter="" default="44100">
-			The mixing rate used for audio (in Hz). In general, it's better to not touch this and leave it to the host operating system.
+			Target mixing rate used for audio (in Hz). In general, it's better to not touch this and leave it to the host operating system.
+			[b]Note:[/b] Mixing rate might be overridden by audio driver.
+			[b]Note:[/b] Input and output mixing rates might be different. Use [method AudioServer.get_mix_rate] and [method AudioServer.get_input_mix_rate] to get actual values.
+		</member>
+		<member name="audio/driver/mix_rate.ios" type="int" setter="" getter="" default="0">
+			Override for [member audio/driver/mix_rate] for iOS. On iOS, mixing rate is determined by audio driver, this value is ignored.
```

Comment:
Is the value here really ignored if you set it to non-zero? Because then I'd say exposing these overrides is counterintuitive, and it would be better to document in the main `mix_rate` setting that it's not used by macOS and iOS.

If what you meant is that only with the default value of 0, then the mixing rate is determined by the driver, then I'd rephrase like this:
```suggestion
			Override for [member audio/driver/mix_rate] for iOS. The default value of [code]0[/code] means that the mixing rate is determined by the audio driver.
```

Reply:
It is always ignored.

Reply:
Then I'd suggest to remove the overrides and just document it in the main property that it has no impact for macOS and iOS.

Reply:
Done.